docs/configuration/source/github-events.md

```diff
@@ -0,0 +1,214 @@
+---
+id: github-events
+title: GitHub Events
+sidebar_position: 4
+---
+
+The Botkube GitHub Events source sends events for configured GitHub repositories. These events can be sent to communication channels or actions. To learn how to bind sources to communication channels or actions, refer to the [Communication](../communication/index.md) and [Action](../action.md) documents.
```

Comment:
Does the source [plugin] receive or send events?

If it receives events I'd say something like `...receives events from Github for configured repositories and forwards them to [botkube channels]`

Reply:
so plugin actually polls the events, one are directly from the `/events` REST endpoint and additional ones are "generated" by listing pull requests for a specific repository and based on that we "convert" them into events

but once we will switch to the webhook approach that IMO your statement will be 100% valid